streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -176,7 +176,7 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
       blockGenerator.addData(count)
       generatedData += count
       count += 1
-      Thread.sleep(1)
+      Thread.`yield`()
```

Comment:
Unless I'm about to learn some crazy new syntax, this won't compile. `Thread.yield()`? but if the point is that this loop needs to happen "really fast" then just remove it? I get that this will flood the block generator much much more rapidly, but that might have already been happening with `yield()`. In any event, if you just want to wait less than a millisecond, you can call `Thread.sleep(0, [# nanoseconds])`.